charts/rancher-pushprox/102.0.0/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+annotations:
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.25.0-0'
```

Comment:
I think this needs to be updated.

Reply:
Good catch. Will update in this and main chart.